Proofread (#33)
MicrosoftDocs/dataexplorer-docs: data-explorer/kusto/api/rest/authentication.md

---
# Authentication over HTTPS

When using HTTPS, the service supports the standard HTTP `Authorization` header
for performing authentication.

> The HTTP authentication methods are:

Reviewer: ```suggestion
The supported HTTP authentication methods are:
```

Reviewer: Why did you take out supported?

* **Azure Active Directory**, via the `bearer` method.

> When authenticating using the Azure AD, the `Authorization` header has

Reviewer: ```suggestion
When authenticating using Azure AD, the `Authorization` header has
```

> When authenticating using the Azure Active Directory, the `Authorization` header has

Reviewer: ```suggestion
When authenticating using Azure AD, the `Authorization` header has
```

the format:

```txt
Authorization: bearer TOKEN
```

Where `TOKEN` is the access token that the caller acquires by communicating with

> the Azure Active Directory service. THe token has the following properties:

Reviewer: ```suggestion
the Azure AD service. The token has the following properties:
```

* The resource is the service URI (e.g., `https://help.kusto.windows.net`)
* The Azure Active Directory service endpoint is
  `https://login.microsoftonline.com/TENANT/`

> Where `TENANT` is the Azure Active Directory tenant ID or name. 

Reviewer: ```suggestion
Where `TENANT` is the Azure AD tenant ID or name. 
```

For example, services that are created under the Microsoft tenant can use
`https://login.microsoftonline.com/microsoft.com/`. 
Alternatively, for user authentication only, the request can be made to
`https://login.microsoftonline.com/common/`.

> [!NOTE]

> > The Azure Active Directory service endpoint changes when it runs in national clouds.

Reviewer: ```suggestion
> The Azure AD service endpoint changes when it runs in national clouds.
```

> To change the endpoint, set an environment variable `AadAuthorityUri` to the required URI.

For more informaton, see the [authentication overview](../../management/access-control/index.md)

> and the [guide to Azure Active Directory authentication](../../management/access-control/how-to-authenticate-with-aad.md).

Reviewer: ```suggestion
and the [guide to Azure AD authentication](../../management/access-control/how-to-authenticate-with-aad.md).
```
